src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,413 @@
+context of the academic literature.
+
+Several [example PET datasets](https://github.com/bids-standard/bids-examples) have been formatted using this specification
+and can be used for practical guidance when curating a new dataset.
+
+Further PET datasets are available from [OpenNeuro](https://openneuro.org).
+
+## Terminology and conventions
+
+PET-BIDS is fully consistent with the BIDS specification as a whole.
+However, BIDS was initially developed in the context of MRI,
+so some terminology may be unfamiliar to researchers from each field.
+This section adds clarifications to
+[Common Principles - Definitions](../02-common-principles.md#definitions)
+for the PET context, and introduces the term "time zero" which is currently
+specific to PET.
+
+1.  **Session** - In most cases, a new session with respect to PET corresponds
+    to a visit to the scanning site, and starts with a new injection.
+    In situations where different data types are obtained over several visits
+    (for example, FDG PET on one day followed by amyloid PET a couple days after)
+    these scans may be grouped into the same session.
+    In other datasets, a subject leaving the scanner and returning under the same
+    injection may be considered separate sessions.
+
+1.  **Run** - In PET, subjects may have to leave the scanner to use the bathroom.
+    While leaving the scanner would interrupt an MR acquisition, in PET this
+    disruption is more appropriately considered missing data during a run.
+
+1.  **Time zero** - A reference point in time, to which all timestamps pertaining
+    to a recording are relative.
+    Time zero will most commonly be the time of injection of a radioisotope, or
+    the time at which the first scan of an acquisition is begun.
+    If a pharmacological within-scan challenge is performed,
+    another time zero may be more convenient.
+
+An overview of a common PET experiment (with blood data) can be seen in Figure 1,
+defined on a single time scale relative to a predefined "time zero".
+
+![Figure 1](images/PET_scan_overview.png "Overview of PET recording data")
+
+Figure 1: Overview of a common PET experiment, including blood measurements,
+and defined on a common time scale.
+Note, "time zero" is often defined as time of injection or scan start, but if a pharmaceutical challenge is carried out during the scan, this timepoint may also be chosen as time zero. The injected dose, the PET data, and blood data should all be decay-corrected to time zero, but because the time of injection not always coincides with scan start, the PET data may not always be decay-corrected to the time of injection. If this is not the case, this may be indicated in the reconstruction section (`ImageDecayCorrected` and `ImageDecayCorrectionTime`). In the example in the figure, tracer injection coincides with scan start.
+and all the PET and blood data should be decay-corrected to this time point.
```

Comment:
Mostly formatting, with "not always coincides" -> "does not always coincide". The second line appears to be a fragment from a previous version, so I removed it.

```suggestion
Note, "time zero" is often defined as time of injection or scan start,
but if a pharmaceutical challenge is carried out during the scan,
this time point may also be chosen as time zero.
The injected dose, the PET data, and blood data should all be decay-corrected to time zero,
but because the time of injection does not always coincide with scan start,
the PET data may not always be decay-corrected to the time of injection.
If this is not the case, this may be indicated in the reconstruction section
(`ImageDecayCorrected` and `ImageDecayCorrectionTime`).
In the example in the figure, tracer injection coincides with scan start.
```

Reply:
Thanks, @effigies !